Docs/ProjectSpec.md

```diff
@@ -354,6 +355,9 @@ targets
         - Staging
         - Production
       gatherCoverageData: true
+      commandLineArguments:
```

Comment:
Most arguments will use “-myArgument” format, so its good to show quotes are needed

Reply:
👍  I can update with quotes

Reply:
PR here: https://github.com/yonaskolb/XcodeGen/pull/176